extensions/formats/stanag4676/format/src/main/java/mil/nga/giat/geowave/format/stanag4676/image/ImageChipDataAdapter.java

```diff
@@ -112,4 +113,43 @@ public void fromBinary(
 		return null;
 	}
 
+	@Override
+	public int getPositionOfOrderedField(
+			final CommonIndexModel model,
+			final ByteArrayId fieldId ) {
+		int i = 0;
+		for (final NumericDimensionField<? extends CommonIndexValue> dimensionField : model.getDimensions()) {
+			if (fieldId.equals(dimensionField.getFieldId())) {
+				return i;
+			}
+			i++;
+		}
+		if (fieldId.equals(IMAGE_FIELD_ID)) {
```

Comment:
this is a single field adapter, as is the rasterdataadapter...basically the field ID is constant and the adapter puts everything in a single field. So this is essentially saying "is the field the one expected value, if not return -1.  Because line 72 of this class in the encode() method just puts a single field with a constant field ID in the encoding, the only field that it should have is that constant field ID